src/Components/SideBar/SideBar.module.scss

```diff
@@ -0,0 +1,94 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap");
+// .............................................................
+
+$border-color: rgb(141, 141, 141, 0.2);
+
+* {
+  margin: 0;
+
+  font-family: poppins;
+}
```

Comment:
This block and the import above is not necessary as index.scss already has these lines, and it was not imported as a module (and hence applies to all pages).

**Always read the index.scss to see variables already defined**

Also, if you want to set a font, keep the name as a variable in index.scss, like --gdsc-font-1 (which already exists, and expands to Poppins)